src/components/sidebar/right-sidebar/CalendarPicker.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useContext, useEffect, useState } from "react";
+import { AppContext } from '../../../context'
+import "./CalendarPicker.css"
+import Translate from "../../../utils/translate/translateService";
+
+import * as moment from 'moment';
```

Comment:
We are currently using another library called date-fns which is more smaller and more frequently updated than moment. moment at this point is considered a deprecated library. 
Check this out: https://momentjs.com/docs/#/-project-status/ for more information on why not to use moment.

The library date-fns is what we are using and you can find the docs here:
https://date-fns.org/v2.16.1/docs/

It's being used in src/utils/utils.tsx. Check it out there for more info!